lib/oedipus_lex.rb

```diff
@@ -474,12 +474,18 @@ def action
 
         attr_accessor :old_pos
 
+        ##
+        # The position of the start of the current line. Only available if the
+        # :column option is on.
+
+        attr_accessor :start_of_current_line_pos
```

Comment:
keep track of the start of the current line. We use this as the off-set of the column position computation.